tests/conftest.py

```diff
@@ -70,6 +73,23 @@
 logger = logging.getLogger(__name__)
 
 
+def skip_if_python_below_minimum_version():
+    """
+    All test fixtures for Rule-Based Profiler must execute this method; for example:
+        ```
+        skip_if_python_below_minimum_version()
+        ```
+
+    This method can also be executed at the module level (by placing the above line at the top, just below the imports).
+
+    for as long as the support for Python versions less than 3.7 is provided.
+    """
```

Comment:
Can you add a note regarding what's breaking? (dataclasses)

Reply:
@cdkini Done.